src/index.js

```diff
@@ -84,7 +84,11 @@ function update(source) {
   nodeUpdate
     .select('circle')
     .attr('r', 4.5)
-    .style('fill', (d) => (d.isDruggable ? '#660066' : '#fff'));
+    .style('fill', (d) => {
+      let color = d.type == undefined ? 
```

Comment:
This could become ` let color = d.type == undefined && d.isDruggable ? colormap[d.parent.parent.parent.name] : '#fff'`